Sources/LeafKit/LeafSerialize/LeafSerializer.swift

```diff
@@ -172,6 +161,18 @@ struct LeafSerializer {
         )
         return try resolver.resolve().map { $0.result }
     }
+    
+    // Directive resolver for a [ParameterDeclaration] where only one parameter is allowed that must resolve to a single value
+    private func resolveAtomic(parameters: [ParameterDeclaration]) throws -> LeafData {
```

Comment:
New internal resolver for fixed-pattern `[ParameterDeclaration]` where one and exactly one Parameter is required, returning the resolved atomic LeafData or null if un-resolvable and throwing if count is wrong.